mergify_engine/web/__init__.py

```diff
@@ -255,6 +255,23 @@ async def queues_by_owner_id(owner_id):
     return responses.JSONResponse(status_code=200, content=queues)
 
 
+@app.get(
+    "/queues_v2/{owner_id}",  # noqa: FS003
+    dependencies=[fastapi.Depends(auth.signature)],
+)
+async def queues(owner_id):
+    global _AREDIS_CACHE
+    queues = collections.defaultdict(dict)
+    async for queue in _AREDIS_CACHE.scan_iter(match=f"merge-queue~{owner_id}~*"):
+        _, _, repo_id, branch = queue.split("~")
+        async with github.aget_client(owner_id=owner_id) as client:
+            queues[client.auth.owner + "/" + repo_id][branch] = [
+                int(pull) async for pull, _ in _AREDIS_CACHE.zscan_iter(queue)
+            ]
```

Comment:
I don't think we want to make a client here to return the owner, it'd be enough to return the repo id and that's it.
The caller knows the `owner_id` since it is sending it.
Thoughts @sileht?

Reply:
Agreed.

Reply:
Do you mean just `queues[repo_id][branch]` ?

Reply:
Yes 👍

Reply:
ok